plugin/plugin.go

```diff
@@ -78,30 +79,35 @@ func (l *Plugin) SendEvent(tag event.Tag, data event.Event) {
 	}
 }
 
-func (l *Plugin) Events(t event.Tag) []event.Event {
+func (l *Plugin) Events(t event.Tag) {
```

Comment:
This function replaces the original Events from Gio, using Unsafe. It must change and do the same for 0.6. In that case, accept the filter. Internally, we need to have our own filter-match.